### DISCUSSION TITLE
Can I apply a custom envinroment image and keep the neutral lighting?

### DISCUSSION THREAD

**Discussion by leoncvlt** [google/model-viewer#2965] [Q&A]
I see in the documentation that the `skybox-image` parameter controls both the environment image and lighting, while `environment-image` controls the lighting only without changing the background image. Is there a way to keep the neutral lighting you'd get by setting `environment-image="neutral"` but change the environment image around the model?

**Comment by samaneh-kazemi** [comment]
Hi @leoncvlt, do you mean you want to have an environment image, but your model reflects as if there is no environment image? If that's the case, I don't think that's possible as we design it to match the physics in real world. Also, make sure to check out our [examples](https://modelviewer.dev/examples/lightingandenv/#neutralLighting) for this.
